flask_admin/babel.py

```diff
@@ -4,11 +4,11 @@
 except ImportError:
 
     def gettext(string, **variables):
-        return string % variables
+        return string if not variables else string % variables
```

Comment:
This is based directly on [flask-babel implementation](https://github.com/python-babel/flask-babel/blob/master/flask_babel/__init__.py#L639).